.gitignore

```diff
@@ -6,3 +6,5 @@ build
 .eggs
 
 /test/e2e/actual_output
+test/java/.gradle/
```

Comment:
please change to `.gradle` and move to other relatives paths under `.eggs`

Reply:
"move to other relatives paths under .eggs"

not sure what you mean by that?